scripts/ci/maven_release.sh

```diff
@@ -4,23 +4,22 @@ PS4="\[\e[35m\]+ \[\e[m\]"
 set -vex
 pushd "$(dirname "${BASH_SOURCE[0]}")/../../"
 
-
 # Use full history for release
-git checkout -B "${TRAVIS_BRANCH}"
-# Add email to link commits to user
+git checkout -B "${BRANCH_NAME}"
+# Define git identity for commits
 git config user.email "${GIT_EMAIL}"
+git config user.name "${GIT_USERNAME}"
```

Comment:
as per below review comment

Reply:
I replaced it with [configure-git-author action](https://github.com/Alfresco/alfresco-build-tools/blob/master/.github/actions/configure-git-author/action.yml).